src/Field/Image.php

```diff
@@ -60,11 +60,12 @@ public function process($field)
     {
         $attachmentId = $this->fetchValue($field);
 
-        $attachment = $this->post->attachment()->find(intval($attachmentId));
+        $attachment = Post::find(intval($attachmentId));
```

Comment:
Are you referring to using `$this->post->find(intval($attachmentId))`?

I didn't have any luck with that.